App/Containers/AddCaptionScreen.js

```diff
@@ -21,7 +22,7 @@ class AddCaptionScreen extends React.Component {
   static navigationOptions = ({ navigation }) => {
     const params = navigation.state.params || {}
     return {
-      headerTitle: undefined,
+      headerTitle: 'Photo Caption',
```

Comment:
Based on some feedback watching people onboard, they were confused by the large title hanging under the back arrow. they wanted to be able to click it, not realizing the form was below it. so for now, i've just moved the title up into the top bar.

Reply:
Makes sense